### PR TITLE
Start managing CI files

### PR DESCRIPTION
**Pull request by DavidS**
This requires a few changes to the pdk-templates from this PR: https://github.com/puppetlabs/pdk-templates/pull/293

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,14 @@
 ---
 os:
   - linux
-  # OSX Only tests on the latest Puppet Gem, not the full matrix as there's no need to double up
```
Comment (glennsarti) [inline .travis.yml]: CAn probably move this comment into .sync.yml

**Comment by DavidS** [comment]
@glennsarti I don't see how this PR would disable the testing - I'll have a more in-depth look later, but would appreciate any insight you have

**Comment by glennsarti** [comment]
@DavidS It looks like the stages bit is "breaking" it. There's no `CHECK` environment variable.

**Comment by DavidS** [comment]
@glennsarti that seems to be a problem unrelated to this PR. Looking at a previous run from https://travis-ci.org/puppetlabs/puppetlabs-powershell/builds/620050367 the exactly same behaviour is there. Might need a separate ticket then.

**Comment by glennsarti** [comment]
@DavidS Ok so looks like https://github.com/puppetlabs/puppetlabs-powershell/pull/268 broke the testing 😢

IMO that needs to be rectified before this PR can be merged.  However I'm not a primary maintainer so 🤷‍♂

**Comment by DavidS** [comment]
Due to travis' propensity to multiply cells within a stage, I've now removed the main `os` key and instead put a singular `osx` cell into the acceptance stage (due to its duration). Now travis is testing on OS-X as initially planned.

We can revisit this when moving over to litmus.

**Comment by DavidS** [comment]
@glennsarti are you happy with this solution?

**Pull request merged**